taui/src/taui.css

```diff
@@ -39,7 +48,7 @@
   width: 414px;
   top: 0;
   left: 0;
-  height: 100%;
+  height: 1000px;
```

Comment:
Fixing the `Dock` component height for now. This should be addressed in a styling pass.